.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+| Q | A
+| --- | ---
+| Fixed tickets | fixes #issuenum
+| License | MIT
+
+#### What's in this PR?
+
+Explain the contents of the documentation change.
```

Comment:
I would use the same wording for this and the next section as in https://github.com/sulu-io/sulu/blob/develop/.github/PULL_REQUEST_TEMPLATE.md